packages/contracts/src/utils/JwtVerifier.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "../interfaces/IJwtGroth16Verifier.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { strings } from "solidity-stringutils/src/strings.sol";
+
+struct EmailProof {
+    string domainName; // Domain name of the sender's email
```

Comment:
Can you mention what the `domainName` in this case is -> `[kid, iss, azp]`?

Reply:
@shreyas-londhe kid, iss, and azp are concatenated as a string with | as the delimiter. I'll update the comment.